chirp/drivers/baofeng_uv17.py

```diff
@@ -65,27 +64,33 @@ def _download(radio):
     radio.status_fn(status)
 
     for block_number in radio.BLOCK_ORDER:
-        block_index = memory_map.index(block_number) + 1
-        start_addr = block_index * 0x1000
-        for addr in range(start_addr, start_addr + 0x1000,
-                          radio.BLOCK_SIZE):
-            frame = radio._make_read_frame(addr, radio.BLOCK_SIZE)
-            # DEBUG
-            LOG.debug("Frame=" + util.hexprint(frame))
+        if block_number not in memory_map:
+            """Memory block not found."""
+            raise errors.RadioError('Radio memory is corrupted. \
+                                     Fix this by uploading a backup image \
+                                     to the radio.')
```

Comment:
This won't look like you want, because it includes all the indenting space in front of the other lines. Best to do this:
```
raise errors.RadioError('Radio memory is corrupted. '
                        'Fix this by uploading a backup image '
                        'to the radio.')
```

Reply:
I didn't notice anything strange with the message box. But your solution is how it's supposed to be. thnx